public/app/plugins/datasource/tempo/streaming.ts

```diff
@@ -85,8 +85,8 @@ export function doTempoChannelStream(
           }
 
           frames = [
-            metricsDataFrame(metrics, frameState, elapsedTime),
             ...formatTraceQLResponse(traces, instanceSettings, query.tableType),
+            metricsDataFrame(metrics, frameState, elapsedTime),
```

Comment:
Worth adding a comment or a test to make it clear why this order cannot be changed?